IM_calculation/Advanced_IM/advanced_IM_factory.py

```diff
@@ -1,12 +1,16 @@
+from pathlib import Path
 from collections import namedtuple
+
 import pandas as pd
 import os
 import re
 import subprocess
 import tempfile
 
-from qcore.timeseries import seis2txt
+from qcore.timeseries import timeseries_to_text
```

Comment:
This should just import the timeseries module not the function within the module